tools/protodoc/protodoc.py

```diff
@@ -498,9 +498,14 @@ def FormatFieldAsDefinitionListItem(outer_type_context, type_context, field, pro
         manifest_description.edge_config)
   else:
     formatted_security_options = ''
-
-  comment = '(%s) ' % ', '.join([FormatFieldType(type_context, field)] +
-                                field_annotations) + formatted_leading_comment
+  pretty_label_names = {
+      field.LABEL_OPTIONAL: '',
+      field.LABEL_REQUIRED: 'required ',
```

Comment:
I don't think this does anything, as all fields are technically required in proto3. Instead, we have https://github.com/envoyproxy/envoy/blob/master/tools/protodoc/protodoc.py#L458 for where PGV indicates that the field is required by annotation.

Reply:
Removed required.